src/components/ColorPalette/_color-palette.scss

```diff
@@ -0,0 +1,348 @@
+// MAIN CONTAINER
```

Comment:
In general, most of the new styles we've added with the theme use css modules. Especially for something as compartmentalized as this it not only prevents class collisions but enforces good practices for not cross-referencing styles between sheets. 

Something like `dark-mode` or `light-text` could very easily be used in a different stylesheet and we'd be impacting those styles.